src/js/components/CheckBox/StyledCheckBox.js

```diff
@@ -36,7 +48,11 @@ const StyledCheckBoxContainer = styled.label`
   flex-direction: row;
   align-items: center;
   user-select: none;
-  width: fit-content;
+  ${props =>
+    props.fillProp &&
+    fillStyle(props.fillProp) &&
+    `justify-content: space-between;`} ${props =>
```

Comment:
Do we always want it to justify with space-between? what about if I wanted `fill` so my hover background would show, but the CheckBox/label was still justified to the start

Reply:
Should caller have control of that

Reply:
I was following the pattern of the other PR. In this case yes this would be always set to  ` justify with space-between` however I agree with your point of if you  just wanted the hover background and not the `space-between` Wondering if we can add another prop for adding the `space-between`

Reply:
@ericsoderberghp would you agree to adding another prop that accounts for the `justify`?

Reply:
I would be OK with not adding a `justify` property and just adding `fill` for now. We can add other layout properties later when we have a concrete need, as opposed to a hypothetical one. I think defaulting to `justify-between` behavior is OK as I think that's the 80% case.